@skitt @dfarrell07 e2e is failing for an unrelated error

```
 > [builder 3/3] RUN make -C /go/src/github.com/submariner-io/submariner LOCAL_BUILD=1 bin/linux/amd64/submariner-route-agent:
------
executor failed running [/bin/sh -c make -C ${SOURCE} LOCAL_BUILD=1 bin/${TARGETPLATFORM}/submariner-route-agent]: exit code: 2
make[1]: *** [/opt/shipyard/Makefile.images:24: package/.image.submariner-route-agent] Error 1
make: *** [/opt/shipyard/Makefile.inc:139: e2e] Error 2
[18:12:34.453] [submariner]$ make --debug=b e2e
make: *** [Makefile.dapper:45: e2e] Error 2
Error: Process completed with exit code 2.
```

--- a/pkg/routeagent_driver/handlers/kubeproxy/vxlan.go
+++ b/pkg/routeagent_driver/handlers/kubeproxy/vxlan.go
@@ -314,7 +314,7 @@ func (kp *SyncHandler) ensureLooseModeIsConfigured(iface string) error {
 				return nil
 			}
 		} else {
-			logger.Warningf("Error retrieving reverse path filter for %q: %v", iface, err)
+			klog.Warningf("Error retrieving reverse path filter for %q: %v", iface, err)
 		}
 
 		err = kp.netLink.EnableLooseModeReversePathFilter(iface)

